src/defaultPropTypes.js

```diff
@@ -17,11 +17,11 @@ export default {
   activeBarStyle: PropTypes.object,
   dismissAfter: PropTypes.oneOfType([
     PropTypes.bool,
-    PropTypes.number
+    PropTypes.number,
```

Comment:
@BerkeleyTrue Dangling commas are part of the airbnb styleguide - they were added when I linted the project so it would pass. I prefer to go without, but what do you guys think?